{"name": "Prueba de login fallido", "status": "passed", "description": "Verifica que el login falle con credenciales incorrectas", "attachments": [{"name": "stdout", "source": "3877c0e5-2244-46f2-9944-523f35d6f13d-attachment.txt", "type": "text/plain"}], "start": 1747669697259, "stop": 1747669697355, "uuid": "7a027cb1-68af-416c-983f-8b464e7f5ef1", "historyId": "f892cd1ad0909e3cab6ddca4d9ed5d9b", "testCaseId": "f892cd1ad0909e3cab6ddca4d9ed5d9b", "fullName": "EcoGuardian.tests.test_eco_guardian.AuthTests#test_login_fallido", "labels": [{"name": "severity", "value": "critical"}, {"name": "parentSuite", "value": "EcoGuardian.tests"}, {"name": "suite", "value": "test_eco_guardian"}, {"name": "subSuite", "value": "AuthTests"}, {"name": "host", "value": "EdwinFigueroa10"}, {"name": "thread", "value": "24596-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "EcoGuardian.tests.test_eco_guardian"}]}